source/apiVolontaria/apiVolontaria/tests/tests_view_ResetPassword.py

```diff
@@ -270,40 +271,93 @@ def test_create_new_token_without_email_service(self, imailing):
             "FRONTEND_INTEGRATION": {
                 "FORGOT_PASSWORD_URL": "fake_url",
             }
-        }
+        },
     )
-    @mock.patch('apiVolontaria.views.IMailing')
-    def test_create_new_token_with_failure_on_email_service(self, imailing):
+    def test_create_new_token_with_failure_email(self):
```

Comment:
What the difference with the second test just below ? `test_create_new_token_with_call_exceptionl`